rfcs/SchemaCoordinates.md

```diff
@@ -0,0 +1,310 @@
+# RFC: Schema Coordinates
+
+**Proposed by:** [Mark Larah](https://twitter.com/mark_larah) - Yelp
+
+This RFC proposes formalizing "Schema Coordinates" - a human readable syntax to
+uniquely identify a type, field, or field argument defined in a GraphQL Schema.
```

Comment:
Should enum value coordinates be addressed by this RFC too? Similar to the use cases discussed for fields and field arguments, we (Salsify) have tooling to understand how clients are using enum values (in addition to fields and field arguments) to support schema evolution. It should fit in pretty well with the proposed dot syntax (e.g. `Direction.NORTH` refers the `NORTH` enum value in the `Direction` enum) so hopefully it's not a big lift.

Reply:
yep - makes sense! thanks for this, incorporated this into the selector grammar.